Helper/Data.php

```diff
@@ -1793,7 +1793,11 @@ public function parseTransactionId($transactionId)
         }, ARRAY_FILTER_USE_KEY);
     }
 
-    public function padShopperReference($shopperReference)
+    /**
+     * @param $shopperReference
+     * @return string
+     */
+    public function padShopperReference($shopperReference): string
```

Comment:
Can we also add the argument type?
```suggestion
    public function padShopperReference(string $shopperReference): string
```